sandboxed_containers/understanding-sandboxed-containers.adoc

```diff
@@ -8,32 +8,51 @@ toc::[]
+* Workloads that require special capabilities from the kernel beyond the default ones granted by standard container runtimes such as CRI-O, for example to access low-level networking features.
+* Workloads that need elevated (root) privileges, for example to access a specific physical device. With {sandboxed-containers-first}, the specific device they use (and only that device) can be passed through to the VM, ensuring that the workload cannot access or misconfigure the rest of the system.
+* Workloads for installing or using `set-uid` root binaries. These binaries grant special privileges and, as such, can present a security risk. With {sandboxed-containers-first}, additional privileges are restricted to the virtual machines, and grant no special access to the cluster nodes.
+
+Some workloads may require privileges specifically for configuring the cluster nodes. Such workloads should still use privileged containers, because running on a virtual machine would prevent them from functioning.
+
+Ensure kernel isolation for each workload:: {sandboxed-containers-first} supports workloads that require custom kernel tuning (such as `sysctl`, scheduler changes, or cache tuning) and the creation of custom kernel modules (such as `out of tree` or special arguments).
+
+Share the same workload across tenants:: {sandboxed-containers-first} enables you to support multiple users (tenants) from different organizations sharing the same OpenShift cluster. The system also lets you run third-party workloads from multiple vendors, such as container network functions (CNFs) and enterprise applications. Third-party CNFs, for example, may not want their custom settings interfering with packet tuning or with `sysctl` variables set by other applications. Running inside a completely isolated kernel is helpful in preventing so-called "noisy neighbor" configuration problems.
+
+Ensure proper isolation and sandboxing for testing software:: You can use {sandboxed-containers-first} to run a containerized workload with known vulnerabilities or to handle an issue in a legacy application.  This isolation also enables administrators to give developers administrative control over pods, which is useful when the developer wants to test or validate configurations beyond those an administrator would typically grant. Administrators can, for example, safely and securely delegate kernel packet filtering (eBPF) to developers. (Kernel packet filtering requires `CAP_ADMIN` or `CAP_BPF` privileges, and is therefore not allowed under a standard CRI-O configuration, as this would grant access to every process on the Container Host worker node.) Similarly, administrators can grant access to intrusive tools such as SystemTap, or support the loading of custom kernel modules during their development.
```

Comment:
I know we can have compete sentences in parentheses, but it does not seem necessary here. Up to you though! 😁